components/omega/external/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 # Add external packages
 
+include(ExternalProject)
```

Comment:
Doesn't look like you're using `ExternalProject`. Maybe a leftover from a previous dev iteration?